lib/foreman_theme_satellite/documentation.rb

```diff
@@ -82,6 +82,9 @@ module Documentation
     ]
 
     DOCS_GUIDES_LINKS = {
+      'Managing_Content' => {
+        'Products_and_Repositories_content-management' => "#{ForemanThemeSatellite.documentation_root}/managing_content/importing_content_content-management#Products_and_Repositories_content-management",
+      },
```

Comment:
This breaks the flattening below :thinking: 
```
ArgumentError: wrong array length at 0 (expected 2, was 4)
```

Reply:
We need to change
``` ruby
          key_values.concat(*nested_to_flat_k_v(key, v))
        else
          key_values.concat([key, v])
```
to
``` ruby
          key_values.append(*nested_to_flat_k_v(key, v))
        else
          key_values.append([key, v])
```
in the flattening. It should be an array of pairs